docs/_includes/asciidoc-vs-markdown.adoc

```diff
@@ -0,0 +1,259 @@
+----
+a|
+[source, asciidoc]
+----
+**b**old
+----
+
+| Italic (word)
+a|
+[source, markdown]
+----
+*italic*
+----
+a|
+[source, asciidoc]
+----
+_italic_
+----
+
+| Italic (first letter)
+|n/a
+a|
+[source, asciidoc]
+----
+__i__talic
+----
+
+| Literal (word)
+a|
+[source, markdown]
+----
+`literal`
+----
+a|
+[source, asciidoc]
+----
++literal+ or `literal`
+----
+
+| Literal (first letter)
+a|
+[source, markdown]
+----
+`l`iteral
+----
+a|
+[source, asciidoc]
+----
+++l++iteral
+----
+
+| Link with text
+a|
+[source, markdown]
+----
+[AsciiDoc](http://asciidoc.org)
+----
+a|
+[source, asciidoc]
+----
+http://asciidoc.org[AsciiDoc]
+----
+
+| Image w/ Alt Text
+a|
+[source, markdown]
+----
+![Logo](/images/logo.png)
+----
+a|
+[source, asciidoc]
+----
+image:logo.png[Logo]
+----
+
+| Headings
+a|
+[source, markdown]
+----
+## Heading 2
+----
+a|
+[source, asciidoc]
+----
+== Heading 2
+----
+
+| Blockquote
+a|
+[source, markdown]
+----
+> Quoted text
+> with two paragraphs
+----
+a|
+[source, asciidoc]
+----
+[quote]
+____
+Quoted text
+
+With two paragraphs
+____
+----
+| Code blocks
+a| (GitHub-flavored Markdown)
+[source, markdown]
+----
+```java
+public class Person {
+}
+```
+----
+a|
+[source, asciidoc]
+----
+[source,java]
+--
+public class Person {
+}
+--
+----
+
+| Unordered list
+a|
+[source, markdown]
+----
+* apples
+* orange
+  * temple
+  * navel
+* bananas
+----
+a|
+[source, asciidoc]
+----
+* apples
+* oranges
+** temple
+** navel
+* bananas
+----
+| Ordered list
+a|
+[source, markdown]
+----
+1. first
+2. second
+3. third
+----
+a|
+[source, asciidoc]
+----
+. first
+. second
+. third
+----
+|===
+
+You can see that AsciiDoc has the following advantages over Markdown:
+
+* it uses the same number of markup characters or less when compared to Markdown in nearly all cases
+* it employs a consistent formatting scheme
+* it can handle all permutations of nested inline (and block) formatting, whereas Markdown often falls down
+* it handles cases that Markdown doesn't (like a proper approach to inner-word markup and source code blocks)
++
+For example:
++
+*Definition Lists in AsciiDoc*
++
+[source, asciidoc]
+--
+a term:: a definition
+another term:: another definition
+--
++
+They can even hold code examples:
++
+[source, asciidoc]
+--
+term with code example::
+description
++
+[source,java]
+public class Person {
+}
+--
++
+*Tables in AsciiDoc*
++
+An AsciiDoc table can be written as a series of lists which use a vertical bar as the list marker:
++
+[source, asciidoc]
+--
+[cols=3]
+|===
+| a
+| b
+| c
+
+| 1
+| 2
+| 3
+|===
+--
++
+Which appears as:
++
+[cols=3]
+|===
+| a
+| b
+| c
+
+| 1
+| 2
+| 3
+|===
++
+Markdown Extra supports tables and definition lists; but it’s not Markdown.
+Also, unlike Markdown Extra, Asciidoc can apply formatting to cells.
+
+Asciidoctor, which is used on GitHub, also emulates "`the good parts`" of the Markdown syntax, like headings, blockquotes and fenced code blocks: migration is fairly easy.
```

Comment:
Let's add a reference to the Markdown support. For now, let's link to http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#markdown-compatibility